recipe/meta.yaml

```
@@ -1,34 +1,44 @@
-{% set version = "0.4.7.1" %}
+{% set name = "msgpack-numpy" %}
+{% set version = "0.4.8" %}
 
 package:
-  name: msgpack-numpy
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/m/msgpack-numpy/msgpack-numpy-{{ version }}.tar.gz
-  sha256: 7eaf51acf82d7c467d21aa71df94e1c051b2055e54b755442051b474fa7cf5e1
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c667d3180513422f9c7545be5eec5d296dcbb357e06f72ed39cc683797556e69
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - numpy >=1.9.0
+    - numpy {{ numpy }}
```

It is a run dependency:
```suggestion
    - numpy >=1.9.0
```